CMakeLists.txt

```diff
@@ -463,6 +464,31 @@ IF(PLUSBUILD_USE_OpenCV)
   MESSAGE(STATUS "------")
   MESSAGE(STATUS "")
 ENDIF()
+
+# Infrared Thermal Seek camera
+IF(PLUS_USE_INFRARED_SEEK_CAM)

```

Comment:
If possible, please replace with find module for libusb.

(repeat from PlusLib comment)